recipes-devtools/clang/clang/0029-llvm-Recognize-yoe-and-poky-as-OE-distro.patch

```diff
@@ -17,13 +22,11 @@ diff --git a/llvm/lib/Support/Triple.cpp b/llvm/lib/Support/Triple.cpp
 index 4f483c965282..abd181759786 100644
 --- a/llvm/lib/Support/Triple.cpp
 +++ b/llvm/lib/Support/Triple.cpp
-@@ -490,6 +490,9 @@ static Triple::VendorType parseVendor(StringRef VendorName) {
+@@ -490,6 +490,6 @@ static Triple::VendorType parseVendor(StringRef VendorName) {
      .Case("mesa", Triple::Mesa)
      .Case("suse", Triple::SUSE)
-     .Case("oe", Triple::OpenEmbedded)
-+    .Case("yoe", Triple::OpenEmbedded)
-+    .Case("poky", Triple::OpenEmbedded)
-+    .Case("wrs", Triple::OpenEmbedded)
+-    .Case("oe", Triple::OpenEmbedded)
++    .Case("oe", Triple::OpenEmbedded)CLANG_EXTRA_OE_VENDORS_CASES
```

Comment:
Lets make it `//CLANG_EXTRA_OE_VENDORS_CASES` so the code can keep compiling outside OE builds.